02-peer-protocol.md

```diff
@@ -793,10 +793,11 @@ A sending node:
     transaction at the current `feerate_per_kw` while maintaining its channel
     reserve (see [Updating Fees](#updating-fees-update_fee)).
     - SHOULD NOT offer `amount_msat` if, after adding that HTLC to its commitment
-    transaction, its remaining balance doesn't allow it to pay the fee to receive a
-    future additional non-dust HTLC while maintaining its channel reserve ("fee
-    spike buffer"). A buffer that expects the current `feerate_per_kw` to potentially
-    double is recommended to ensure predictability between implementations.
+    transaction, its remaining balance doesn't allow it to pay the commitment
+    transaction fee when receiving a future additional non-dust HTLC while maintaining
```

Comment:
> `when receiving a future additional non-dust HTLC`

maybe:  `when adding another potential non-dust HTLC`

Reply:
or just: `when adding a future additional non-dust HTLC`
Or does the word `receiving` add anything important here?

Reply:
I think `receiving` is important: see that discussion: https://github.com/lightningnetwork/lightning-rfc/pull/740#discussion_r399197774